pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -307,6 +309,7 @@ def test_merge_single_cells():
                 pd.testing.assert_frame_equal(
                     norm_method_df.sort_index(axis=1),
                     manual_merge_normalize.sort_index(axis=1),
+                    check_dtype=False
```

Comment:
This is a bit concerning to me. Can you report what changed and why ignoring column dtype is now necessary?

Reply:
In order to read in features faster, I load them into a `np.array` (which I initialize with `dtype=np.float64`). Original metadata might be a `np.int64`, but this should not matter.

Reply:
sounds good!